docker/Dockerfile

```diff
@@ -0,0 +1,30 @@
+FROM python:3.9-slim-bullseye as parent
```

Comment:
> you need to host that docker image somewhere

Well, we have ghcr already for loads of our images.

> how does a local dev build one of the docker images

One of the sub images (eg antivirus)? Normally I would think; it will just pull from ghcr (we do this for some apps already I'm sure... yep, eg antivirus)

Reply:
Yep, could do GHCR.

I'm not sure, I'm currently thinking that we will still have some use of GHCR for a while at least. 

If we put some of our sub images in ECR, it can make it a bit harder to pull them because we currently put images in private image repositories. (maybe we just explore their public image repositories)